src/goose/cli/main.py

```diff
@@ -114,15 +114,16 @@ def get_session_files() -> dict[str, Path]:
 @click.option("--profile")
 @click.option("--plan", type=click.Path(exists=True))
 @click.option("--log-level", type=click.Choice(["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"]), default="INFO")
-def session_start(name: Optional[str], profile: str, log_level: str, plan: Optional[str] = None) -> None:
+@click.option("--safe-mode", is_flag=True, help="Prompt before executing potentially unsafe commands", default=False)
```

Comment:
wonder if we can borrow from other tools. I think sometimes, if prompt is default, `-Y` is overriding it, like apt

```
       -y, --yes, --assume-yes
           Automatic yes to prompts; assume "yes" as answer to all prompts and run
           non-interactively. If an undesirable situation, such as changing a held package,
           trying to install a unauthenticated package or removing an essential package occurs
           then apt-get will abort. Configuration Item: APT::Get::Assume-Yes.

       --assume-no
           Automatic "no" to all prompts. Configuration Item: APT::Get::Assume-No.
```

Usually, when choosing to name things, I try to borrow from another tool's name that's well used, whether it is vocab or a flag, then you can blame it on them :)

Reply:
haha, naming is always the hardest part to me.. Thank you so much for your advice!